src/components/ProbeStatus.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC, useState } from 'react';
+import { css } from 'emotion';
+import {
+  Badge,
+  BadgeColor,
+  Button,
+  Container,
+  ConfirmModal,
+  Legend,
+  IconName,
+  stylesFactory,
+  useTheme,
+} from '@grafana/ui';
+import { Probe, OrgRole } from 'types';
+import { SMDataSource } from 'datasource/DataSource';
+import { hasRole } from 'utils';
+import { UptimeGauge } from './UptimeGauge';
+import { GrafanaTheme } from '@grafana/data';
+
+interface Props {
+  probe: Probe;
+  instance: SMDataSource;
+  onResetToken: () => void;
+}
+
+interface BadgeStatus {
+  color: BadgeColor;
+  text: string;
+  icon: IconName;
+}
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  legend: css`
+    margin: ${theme.spacing.sm} ${theme.spacing.sm} 0 0;
+    width: auto;
+  `,
+  container: css`
+    padding-left: ${theme.spacing.sm}
+    margin-bottom: ${theme.spacing.md}
```

Comment:
Does that work? Because you're missing semicolons here and IIRC, emotion isn't quite so smart to put them in anyway.
If you use vscode, I'd recommend installing the extension `vscode-styled-components` because that would've thrown an error. (And it provides syntax highlighting :) ).